run_docker

```diff
@@ -137,6 +137,7 @@ CONTAINER=$(\
         --groups sudo  \
         $USER_NAME \
       ; \
+      echo 'export PATH=\$PATH:$USER_HOME/.local/bin' >> /etc/profile; \
```

Comment:
This is to make `pyodide` entrypoint (and other python applications) availble in the docker image.